### DISCUSSION TITLE
"Copy code" button is annoying to reach

### DISCUSSION THREAD

**Discussion by Fusseldieb** [open-webui/open-webui#4818] [Ideas]
**Is your feature request related to a problem? Please describe.**
When copying long code, I always need to scroll all the way up to hit the "Copy code" button. This becomes especially tedious when dealing with long code, and long debugging sessions.

**Describe the solution you'd like**
Ideally, the "Copy code" button should stay within the visible boundaries of the code window. 

**Describe alternatives you've considered**
\-

**Additional context**
Anthropic does this in their API workbench, for example.

**Comment by Keithsc** [comment]
Thanks for the discussion on this feature request! I wanted to clarify that having the "Copy code" button at the bottom of the code block is a good start, but I believe it would be even more beneficial to keep the button within the visible boundaries of the code window as you scroll. 

When dealing with long blocks of code, it's crucial that the button remains accessible without any need to scroll back up and down. This would enhance usability significantly during coding and debugging sessions.

Looking forward to seeing how this can be implemented!
